modules/apps/portal-remote/portal-remote-json-web-service-web/src/main/java/com/liferay/portal/remote/json/web/service/web/internal/util/ExtractedParameter.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.remote.json.web.service.web.internal.util;
+
+/**
+ * @author Tamas Biro
+ */
+public class ExtractedParameter {
```

Comment:
Hey @jgarcialr, Can you try to get rid of this class and use the  `MethodParameter` class directly? 

Thx

/cc @tomibiro

Reply:
Once you finish this change, I'll review the pr completely. 

Also, please add a unit test for the `MethodParameterExtractor#getMethodParameters`.

Thanks in advance